src/dmclock/sim/src/sim_recs.h

```diff
@@ -105,7 +105,7 @@ namespace crimson {
     struct TestResponse {
       uint32_t epoch;
 
-      TestResponse(uint32_t _epoch) :
```

Comment:
please send a PR to https://github.com/ceph/dmclock instead.

Reply:
I can do that. Is there any reason why dmclock is part of the ceph.git tree and not a submodule? It's really confunsing to have code from a separate repo that is copied into the tree.

Reply:
I know there is a README from commit e16344644e91cb2cb28024c1ddb5832758a4cb8d but this will not help at all. Would you accept a change to make this a submodule?

Reply:
this topic was discussed at https://www.spinics.net/lists/ceph-devel/msg36019.html 1 year ago. if you want to change subtree to submodule, probably you could start a discussion at ceph-devel first. it's not my call to decide whether we should use submodule or subtree for including dmclock.